### DISCUSSION TITLE
API not Listening on port 5000 on windows?

### DISCUSSION THREAD

**Discussion by B33T** [oobabooga/text-generation-webui#1607] [Q&A]
I have enabled API and if I use postman etc I can use http://localhost:5000/api/v1/generate, but if I try to use my LAN IP address, it fails to connect example http://192.168.178.81:5000/api/v1/generate 
When I look in window if i open resource monitor it doesn't list as listening on port 5000, If I do a check with my firewall it states no connection(even with firewall turned off), the devices can ping each other and the web portal does work on http://192.168.178.81:7860 but for some reason not the API any ideas?

**Comment by corvec** [comment]
What command line parameters are you passing?
